lumigator/python/mzai/backend/backend/tracking/schemas.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+class RunOutputs(BaseModel):
```

Comment:
Does it make sense to just use dicts to Any?

Reply:
I was hoping to keep that specifying the metrics dict needs to have float values, since we do some basic sigfig rounding before sending to the client (I think right now I round to 3)

Reply:
Ok